packages/vm/package.json

```diff
@@ -69,7 +69,6 @@
     "async-eventemitter": "^0.2.4",
     "debug": "^4.3.3",
     "ethereum-cryptography": "^1.1.2",
-    "functional-red-black-tree": "^1.0.1",
```

Comment:
Oh. Seems this was simply forgotten after the state manager extraction work. 🙂 

Good catch! 👍